src/coreclr/jit/assertionprop.cpp

```diff
@@ -168,6 +168,25 @@ bool IntegralRange::Contains(int64_t value) const
             }
             break;
 
+        case GT_IND:
+        {
+#if defined(_WIN64)
```

Comment:
The right check would be `#ifdef WINDOWS_AMD64_ABI`, but I would honestly just leave it out, looks like this doesn't have measurable impact on throughput from SPMI.